config/routes.rb

```diff
@@ -121,6 +121,7 @@
 
   root to: "home#index"
 
+  get "/" => "home#index", :as => :home
```

Comment:
home#index can already be achieved by using the `root_url` defined above this line @bkmaibach